frontend/src/citizen-frontend/calendar/CalendarGridView.tsx

```diff
@@ -327,29 +491,32 @@ const DayCell = styled.button<{
   min-height: 150px;
   padding: ${defaultMargins.s};
   background-color: ${(p) =>
-    p.holidayPeriod
+    p.markedByEmployee
+      ? p.theme.colors.grayscale.g15
+      : p.holidayPeriod
       ? p.theme.colors.accents.a10powder
       : p.theme.colors.grayscale.g0};
   border: none;
   outline: 1px solid ${colors.grayscale.g15};
   cursor: pointer;
   user-select: none;
+  text-align: left;
 
   ${(p) =>
     p.today
       ? css`
           border-left: 4px solid ${colors.status.success};
           padding-left: calc(${defaultMargins.s} - 3px);
         `
-      : ''}
+      : ''};
 
   ${(p) =>
     p.selected
       ? css`
           box-shadow: 0 2px 3px 2px #00000030;
           z-index: 1;
         `
-      : ''}
+      : ''};
```

Comment:
Redundant semicolons? 🤔

Reply:
Yeah kinda, but IntelliJ IDEA's autoformat breaks without them and styled-components seems to handle them correctly.